src/constants.rs

```diff
@@ -1,4 +1,5 @@
 pub const FUELUP_PATH: &str = ".fuelup";
```

Comment:
```suggestion
pub const FUELUP_DIR: &str = ".fuelup";
```

Maybe worth clarifying that this is the directory name to avoid confusion with fuelup exe path?